frontend/src/core/observations/ObservationAssessment.tsx

```diff
@@ -114,7 +114,7 @@ const ObservationAssessment = () => {
                                 choices={OBSERVATION_VEX_JUSTIFICATION_CHOICES}
                             />
                         )}
-                        <TextInputWide source="comment" validate={validate_required_255} />
+                        <TextInputWide multiline={true} source="comment" validate={validate_required} />
```

Comment:
```suggestion
                        <TextInputWide multiline={true} source="comment" validate={validate_required} minRows={3} />
```

and see remark about the size limitiation in the model